docs/user/openstack/README.md

```diff
@@ -44,6 +47,69 @@ enough to store the ignition config files, so they are served by swift instead.
 * You may need to increase the security group related quotas from their default
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
 
+
+* A file `install-config.yaml` is required. It defines your cluster topology. Save it because a failed installation will delete it. It looks like this:
+
+```
+apiVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.master
+  replicas: 3
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.master
+  replicas: 1
+metadata:
+  name: test-cluster
+networking:
```

Comment:
I think everything under `networking` has reasonable defaults that I'd like to preserve.

Leaving the `machineCIDR` option in makes sense (that's the Neutron subnet range for the Nova VMs), but I'd like to remove everything else.